crates/arroyo-operator/Cargo.toml

```diff
@@ -18,6 +18,7 @@ ahash = "0.8"
 async-trait = "0.1.68"
 bincode = "2.0.0-rc.3"
 datafusion = "35.0"
+datafusion-common = "35.0"
```

Comment:
Do we need this since we already have datafusion as a dependency?

Reply:
No, can use datafusion::common.